v4/export/sql_type.go

```diff
@@ -47,11 +45,23 @@ var dataTypeBin = []string{
 	"BIT",
 }
 
-func escape(s []byte, bf *bytes.Buffer, escapeBackslash bool) {
-	if !escapeBackslash {
-		bf.Write(bytes.ReplaceAll(s, quotationMarkNotQuote, quotationMarkQuote))
+func getEscapeQuotation(escapeBackSlash, isCsv bool) byte {
```

Comment:
set the input to the quotation mark (a `byte`) rather than `isCsv`.

in general it's not a good idea to have many bool parameters.